setup.md

```diff
@@ -7,18 +7,18 @@ title: Setup
 > Data for this lesson is from the Portal Project Teaching Database -
 > [available on FigShare](https://figshare.com/articles/Portal_Project_Teaching_Database/1314459).
 >
-> We will use the eight files listed below for the data in this lesson.  Download these files to your computer either by clicking [this link ](https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/datacarpentry/python-ecology-lesson/tree/gh-pages/data), which will give you everything in a single compressed file.  You'll need to unzip this file after downloading it.
+> We will use the eight files listed below for the data in this lesson.  Download these files to your computer either by clicking [this link](https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/weecology/portal-teachingdb), which will give you everything in a single compressed file.  You'll need to unzip this file after downloading it.
 >
 > Or download each file indvidually with the following links:
 >
-> - [surveys.csv]({{ page.root }}/data/surveys.csv)
-> - [species.csv]({{ page.root }}/data/species.csv)
+> - [surveys.csv](https://ndownloader.figshare.com/files/10717177)
+> - [species.csv](https://ndownloader.figshare.com/files/3299483)
 > - [speciesSubset.csv]({{ page.root }}/data/speciesSubset.csv)
 > - [surveys2001.csv]({{ page.root }}/data/survey2001.csv)
 > - [surveys2002.csv]({{ page.root }}/data/survey2002.csv)
-> - [plots.csv]({{ page.root }}/data/plots.csv)
+> - [plots.csv](https://ndownloader.figshare.com/files/3299474)
 > - [bouldercreek_09_2013.txt]({{ page.root }}/data/bouldercreek_09_2013.txt)
-> - [SQL Database]({{ page.root }}/data/portal_mammals.sqlite)
+> - [SQL Database](https://ndownloader.figshare.com/files/11188550)
```

Comment:
@wrightaprilm, do we want to keep copies of the data files in this repo (in the `data` folder)?